radicle-cli/src/commands/id.rs

```diff
@@ -21,11 +22,11 @@ Usage
 
     rad id (update|edit) [--title|-t] [--description|-d]
                          [--delegates <did>] [--threshold <num>]
-                         [--no-confirm]
-    rad id list
-    rad id (show|rebase) <id> [--rev <revision id>]
-    rad id show <id> [--rev <revision id>] [--revisions]
-    rad id (accept|reject|close|commit) [--rev <revision id>] [--no-confirm]
+                         [--no-confirm] [<option>...]
+    rad id list [<option>...]
+    rad id rebase <id> [--rev <revision id>] [<option>...]
+    rad id show <id> [--rev <revision id>] [--revisions] [<option>...]
```

Comment:
```suggestion
    rad id show <rid> [--rev <revision-id>] [--revisions] [<option>...]
```
And the one above too.